string startStopRequest
---
bool success
string msg
